includes/discovery/sensors/state/panos.inc.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+$temp = snmp_get($device, "panSysHAMode.0", "-Ovqe", "PAN-COMMON-MIB");
+$cur_oid = '.1.3.6.1.4.1.25461.2.1.2.1.13.0';
+$index = '0';
+
+echo $temp;
```

Comment:
The only thing left I think is to change all these `echo` to `d_echo()`

Reply:
I can remove, this is especially useful for testing but not for the merge perhaps.

Reply:
Remove it then if you like :)